primitives/staking/src/lib.rs

```diff
@@ -215,4 +224,121 @@ pub trait StakingInterface {
 	fn set_current_era(era: EraIndex);
 }
 
+/// The amount of exposure (to slashing) than an individual nominator has.
+#[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Encode, Decode, RuntimeDebug, TypeInfo)]
+pub struct IndividualExposure<AccountId, Balance: HasCompact> {
```

Comment:
The `IndividualExposure` struct is used project-wide only with `others` within an `Exposure`, so shall we just call this struct `OtherExposure` instead? We can then unify a lot of naming, such as renaming things like `expected_individual_exposures` to just `expected_others`.

Reply:
I actually think the name makes sense. In the context of a validator, this exposure set is called `others` since its other than validator's own stake, but in itself, it represents individual exposure of each nominator.